mobile/src/components/common/table/table.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactElement } from 'react';
+import { ViewStyle } from 'react-native';
+import {
+  Cell,
+  Rows,
+  Table as UITable,
+  TableWrapper,
+} from 'react-native-table-component';
+
+import { AppTextStyle } from '~/common/types/types';
+import { ScrollView, View } from '~/components/common/common';
+
+import { styles } from './styles';
+
+type AllowedData = string | number;
+
+type Props<Data extends Record<AllowedData, unknown>> = {
```

Comment:
Just string will be enough, we won't pass objects with numeric keys
```suggestion
type Props<Data extends Record<string, unknown>> = {
```